portfolio/src/main/webapp/index.html

```diff
@@ -58,7 +58,39 @@ <h3 id="hobbies"> Hobbies </h3>
 
       <hr>
 
-      <h3 id="contact"> Contact Me </h3>
+      <h3 class="headers" id="more"> More about me </h3>
+
+      <p>My favorite actor you ask?</p>
+      <img class="photo" src="Images/Jim-1.jpg">
+      <br>
+      <p>Click <a href="https://www.google.com/search?tbm=isch&q=jim+carrey+funny+faces">here</a> to see more funny pictures of Jim Carrey!</p>
+
+      <p class="paragraph-1">Some of my favorite movies: </p>
+      <button onclick="addRandomMovie()">Get</button>
+      <div id="movie-container"></div>
+
+      <p> So what do I do when I'm not waiting for the last Austin Powers to come out? I listen to music. Which one of these genres do you think I don't listen to? If you answer right, you'll get a link to my spotify playlist.</p>
```

Comment:
Is there a new austin powers movie coming out?!?!

Reply:
Haha. Might be a long wait.

Reply:
they say they talk about it every month or so. but it has been in "production" ever since 2005 :(